app/src/pages/OnDeviceDisplay/ProtocolSetup/__tests__/ProtocolSetup.test.tsx

```diff
@@ -305,6 +326,12 @@ describe('ProtocolSetup', () => {
       <div>mock ConfirmAttachedModal</div>
     )
     mockUseDoorQuery.mockReturnValue({ data: mockDoorStatus } as any)
+    mockUseModulesQuery.mockReturnValue({
+      data: { data: [mockHeaterShaker] },
+    } as any)
+    when(mockUseDeckConfigurationQuery).mockReturnValue({
+      data: [mockFixture],
+    } as UseQueryResult<DeckConfiguration>)
```

Comment:
nit, since you're not specifying `calledWith()` in here, you can simplify this:

```suggestion
    mockUseDeckConfigurationQuery.mockReturnValue({
      data: [mockFixture],
    } as UseQueryResult<DeckConfiguration>)
```